doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -1758,6 +1670,12 @@ Tactic notations allow to customize the syntax of tactics. They have the followi
    following table. The last command gives examples of tactics which use
    the corresponding kind of argument.
 
+   .. todo this table is too wide in HTML, I see a horizontal scroll bar.
+      Any ideas on easy ways to make it fit?  The hard way is
```

Comment:
We could remove the "as in" column. There was a issue opened recently saying that it is bogus anyway.

Reply:
I think the `as in` column is useful for newcomers wondering how the scopes differ.  Alternatively, if there's a way to do a reverse-search on the grammar ("show me all the tactics whose grammar mentions ne_ident_list"), that'd be a decent replacement.

Reply:
> I think the `as in` column is useful for newcomers wondering how the scopes differ.

I think you mean to see how the nonterminals are used (they're not scopes, eh?).  How important is that use case given that the second column links to the syntax for the nonterminal, which explains it (or eventually will)?  IMHO not too important.

Perhaps we could have the full grammar as an appendix, although Théo wasn't a fan of the concept earlier.  We don't use things like "ne_ident_list"; that would be shown in notation form, e.g.

![image](https://user-images.githubusercontent.com/1253341/80272793-67e22b00-8681-11ea-8efc-a9dc0237ba02.png)

Reply:
Indeed, they're not scopes. Indeed, I don't think that having the full grammar as an appendix would be useful.

Reply:
If @JasonGross thinks the `as in` column should be restored, then so be it. It's not such a big deal that you have to scroll horizontally to see this last column.

Reply:
And so it shall be.